tasks/85_bet-validation/bet-validation.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @description
+ * Checks text message (second argument) with conditions:
+ * The text messages should contain exactly N unique numbers between 1 and M
+ * (first argument), separated by a comma (,) and/or space(s).
+ * Any other character makes the bet invalid.
+ * @example
+ * // returns [1, 2, 3, 4, 5]
+ * validate_bet('1 2 3 4 5)
+ * @param {Number} N - number of numbers needed
+ * @param {Number} M - max number allowed
+ * @param {String} message
+ * @return {Array}
+ */
+// eslint-disable-next-line camelcase
```

Comment:
Do not use it

Reply:
didn't we discuss it? eslint marks the name of function as an error, what should I do then?